### PR TITLE
Bug-2091996: dual-stack support for SNO

### PR DESCRIPTION
**Pull request by mkowalski**
Fixes: Bug-2091996

Version(s): 4.9+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2091996

### CODE REVIEW

**Comment by mkowalski** [comment]
/cc astoycos
/cc rbbratta

You worked on a previous PR to the same file. Can you please have a look here?

File: networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ After converting to dual-stack, all newly created pods are dual-stack enabled.
 
 [NOTE]
 ====
-A dual-stack network is supported on clusters provisioned on only installer-provisioned bare metal infrastructure.
+A dual-stack network is supported on clusters provisioned on only installer-provisioned bare metal infrastructure as well as Single Node OpenShift clusters.
```
Comment (romfreiman) [inline networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc:14]: Can u check how we call the bm sno in the docs and specify it here in the same manner?

Comment (mkowalski) [reply networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc:14]: All I have found is https://docs.openshift.com/container-platform/4.9/installing/installing_sno/install-sno-installing-sno.html which mentions it as "OpenShift on a single node"

**Comment by mkowalski** [comment]
/cc @ahardin-rh
/cc @bpickard22
/cc @anuragthehatter

Picking you as last reviewers to the OVN-related stuff that I have found. Change applies to 4.9 and later because the first mention of SNO happened in 4.9

File: networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc (hunk 1 above)
Comment (mjpytlak) [inline networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc:14]:
```suggestion
A dual-stack network is supported on clusters provisioned on only installer-provisioned bare metal infrastructure, as well as Single Node OpenShift clusters.
```

**Review by mjpytlak — COMMENTED** [review]
Small update. Otherwise, peer review LGTM.

**Comment by sayjadha** [comment]
@mkowalski: A minor comment, and I'm awaiting the doc preview to complete my review. Please let me know when you have set it up.

**Comment by mjpytlak** [comment]
> @mkowalski: A minor comment, and I'm awaiting the doc preview to complete my review. Please let me know when you have set it up.

Good morning, @sayjadha. This is an external contribution. I do not believe we need a docs preview for external contributions.

**Comment by sayjadha** [comment]
@mkowalski : I have one comment, rest LGTM.

File: networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc [hunk 2]
```diff
@@ -11,7 +11,7 @@ After converting to dual-stack, all newly created pods are dual-stack enabled.
 
 [NOTE]
 ====
-A dual-stack network is supported on clusters provisioned on bare metal infrastructure only.
+A dual-stack network is supported only on clusters provisioned on bare metal infrastructure, as well as single-node OpenShift clusters.
```
[the remark this answers is not in the document]
Comment (mkowalski) [reply networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc]: Done, reworded as suggested

File: networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc [hunk 3]
```diff
@@ -11,7 +11,7 @@ After converting to dual-stack, all newly created pods are dual-stack enabled.
 
 [NOTE]
 ====
-A dual-stack network is supported on clusters provisioned on bare metal infrastructure only.
+A dual-stack network is supported on clusters provisioned on bare metal infrastructure as well as single node OpenShift clusters.
```
Comment (mburke5678) [inline networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc:14]: According to the IBM Style Guide
s/as well as/and

**Comment by mburke5678** [comment]
@mkowalski A couple of nits. Otherwise LGTM

**Comment by mburke5678** [comment]
@mkowalski Would it be more accurate to state: 

_A dual-stack network is supported on clusters provisioned on bare metal, IBM Power infrastructure, and single node OpenShift clusters._

**Pull request merged**